CreateSalesEmployeeTable_1.sql

```diff
@@ -0,0 +1,44 @@
+USE JCAC_CormacF
+DELIMITER $$
+DROP PROCEDURE IF EXISTS CreateSalesEmployeeTable_1 $$
+CREATE PROCEDURE CreateSalesEmployeeTable_1()
+BEGIN
+  START TRANSACTION;
+
+	-- Create the Sales_Employee table
+    CREATE TABLE Sale_Employee (
+	SaleEmployeeID int AUTO_INCREMENT PRIMARY KEY,
+    FirstName Varchar(100),
+    LastName Varchar(100),
+    Salary Decimal(10,2),
+    BankAccount varchar(16),
+    NationalInsurance varchar(11)
+	);
```

Comment:
Indentation again. Also be consistent with your use of caps! E.g. varchar and Varchar